### PR TITLE
Fix latency perf jobs

### PR DESCRIPTION
**Pull request by fgelcer**
branch called `branch-perf-v8`now uses AmazonLinux2 images to run performance tests, and it was lacking the 2 commits in here to be able to support it. 

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [ ] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: configurations/manager/debian9.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ami_id_monitor: 'ami-0cfac3931b2a799d1'  # Debian stretch image
```
Comment (fruch) [inline configurations/manager/debian9.yaml:1]: can you this configuration file ? it's not related those changes

File: test-cases/artifacts/amazon2.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+ami_db_scylla_user: 'ec2-user'
```
Comment (fruch) [inline test-cases/artifacts/amazon2.yaml:1]: can you this configuration file ? it's not related those changes

File: configurations/manager/debian9.yaml (hunk 1 above)
Comment (fgelcer) [reply configurations/manager/debian9.yaml:1]: this whole file was marked as "deleted by us", so i "returned" it...
maybe this branch doesn't need it??

File: test-cases/artifacts/amazon2.yaml (hunk 2 above)
Comment (fgelcer) [reply test-cases/artifacts/amazon2.yaml:1]: same as my comment above...

**Comment by fgelcer** [comment]
@fruch please advise

File: configurations/manager/debian9.yaml (hunk 1 above)
Comment (bentsi) [reply configurations/manager/debian9.yaml:1]: "deleted by us" means that your cherry-pick was dependant on the file that doesn't exist in the branch you are backporting to, so it notifies that it was deleted. No need to add it.

File: test-cases/artifacts/amazon2.yaml (hunk 2 above)
Comment (bentsi) [reply test-cases/artifacts/amazon2.yaml:1]: same as the comment above ;)

**Comment by fgelcer** [comment]
removed both files, as they came from the cherry picked commits.

**Comment by fgelcer** [comment]
@bentsi , thank you for the explanation

**Comment by fgelcer** [comment]
job experiences this error now:
`botocore.exceptions.ClientError: An error occurred (InvalidBlockDeviceMapping) when calling the RequestSpotInstances operation: Volume of size 15GB is smaller than  snapshot 'snap-02a9451d6c7ab4d7e', expect size`

IIRC there was a fix from either Evgeniy or Dmitry around this volume size, no?

**Comment by fruch** [comment]
> job experiences this error now:
> `botocore.exceptions.ClientError: An error occurred (InvalidBlockDeviceMapping) when calling the RequestSpotInstances operation: Volume of size 15GB is smaller than snapshot 'snap-02a9451d6c7ab4d7e', expect size`
> 
> IIRC there was a fix from either Evgeniy or Dmitry around this volume size, no?

Yes, something that changed the default size, maybe Amos did it

**Comment by fgelcer** [comment]
> > job experiences this error now:
> > `botocore.exceptions.ClientError: An error occurred (InvalidBlockDeviceMapping) when calling the RequestSpotInstances operation: Volume of size 15GB is smaller than snapshot 'snap-02a9451d6c7ab4d7e', expect size`
> > IIRC there was a fix from either Evgeniy or Dmitry around this volume size, no?
> 
> Yes, something that changed the default size, maybe Amos did it

indeed was Amos.. i found his commit, and pushed here too...

**Comment by roydahan** [comment]
@fgelcer this PR wasn't enough or not good.

The test failed with:

Traceback (most recent call last):
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/cluster.py", line 2798, in node_setup
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR >     cl_inst.node_setup(node, **setup_kwargs)
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/cluster.py", line 3274, in node_setup
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR >     self._scylla_pre_install(node)
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/cluster_aws.py", line 850, in _scylla_pre_install
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR >     self._need_to_install_scylla = not node.is_scylla_installed()
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR >   File "/sct/sdcm/cluster.py", line 1975, in is_scylla_installed
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR >     raise ValueError(f"Unsupported Linux distribution: {self.distro}")
01:44:01  < t:2020-08-18 22:44:00,463 f:cluster.py      l:2800 c:sdcm.cluster         p:ERROR > ValueError: Unsupported Linux distribution: Distro.UNKNOWN

**Comment by fgelcer** [comment]
> @fgelcer this PR wasn't enough or not good.
> 
> The test failed with:
> 
> Traceback (most recent call last):
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > File "/sct/sdcm/cluster.py", line 2798, in node_setup
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > cl_inst.node_setup(node, **setup_kwargs)
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > File "/sct/sdcm/cluster.py", line 3274, in node_setup
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > self._scylla_pre_install(node)
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > File "/sct/sdcm/cluster_aws.py", line 850, in _scylla_pre_install
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > self._need_to_install_scylla = not node.is_scylla_installed()
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > File "/sct/sdcm/cluster.py", line 1975, in is_scylla_installed
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > raise ValueError(f"Unsupported Linux distribution: {self.distro}")
> 01:44:01 < t:2020-08-18 22:44:00,463 f:cluster.py l:2800 c:sdcm.cluster p:ERROR > ValueError: Unsupported Linux distribution: Distro.UNKNOWN

fixed it on #2535

**Pull request merged**